### PR TITLE
sending generation info to datadog

### LINKED ISSUE #3215: Send generation's info to datadog
Each generation's keyblock has an `info` that represents the protocol version of the miner. Expose this to datadog

### PR DESCRIPTION
**Pull request by skkw**
Fixes #3215

### CODE REVIEW

**Comment by dincho** [comment]
What is this about? Link to issue/description ?

File: apps/aecore/src/aec_conductor.erl [hunk 1]
```diff
@@ -1122,6 +1122,10 @@ handle_successfully_added_block(Block, Hash, Events, State, Origin) ->
         {micro_changed, State2 = #state{ consensus = Cons }} ->
             {ok, setup_loop(State2, false, Cons#consensus.leader, Origin)};
         {changed, BlockType, NewTopBlock, State2} ->
+            Origin == block_created andalso
```
Comment (velzevur) [inline apps/aecore/src/aec_conductor.erl:1125]: I see this changes the report to be done only by the miner that produced the block. Why? Do we intend to run this on our non-mining seed nodes?

**Comment by tolbrino** [comment]
@dincho @uwiger @skkw So this is done? At least release notes are missing.

File: apps/aecore/src/aec_conductor.erl (hunk 1 above)
Comment (dincho) [reply apps/aecore/src/aec_conductor.erl:1125]: Hmm, this shouldn't be the case. It should post info for each block that's "appended" to the node database. Including received from sync/p2p gossip.

**Comment by dincho** [comment]
> @dincho @uwiger @skkw So this is done? At least release notes are missing.

No we've review comments above.

**Comment by dincho** [comment]
closing in favour of https://github.com/aeternity/aeternity/pull/3265

**Pull request closed without merging**